tests/e2e/specs/analysis.cy.js

```diff
@@ -265,3 +265,132 @@ describe('Analysis view', () => {
     })
   })
 })
+
+function addView (view) {
+  cy.get('[data-cy=add-view-btn]').click()
+  cy.get(`#toolbar-add-${view}-view`).click()
+    // wait for menu to close
+    .should('not.be.exist')
+}
+
+describe('Filters and Options save state', () => {
+  const numTasks = sortedTasks.length
+  describe('Options save state', () => {
+    it('remembers table and box & whiskers toggle option when switching between workflows', () => {
+      cy.visit('/#/workspace/one')
+      addView('Analysis')
+      cy.get('.c-analysis [data-cy=box-plot-toggle]')
+        .click()
+        .get('.vue-apexcharts')
+        .should('be.visible')
+      // Navigate away
+      cy.visit('/#/')
+      cy.get('.c-dashboard')
+      // Navigate back
+      cy.visit('/#/workspace/one')
+      cy.get('.vue-apexcharts')
+        .should('be.visible')
+    })
+  })
+
+  describe('State saving', () => {
+    beforeEach(() => {
+      cy.visit('/#/workspace/one')
+      addView('Analysis')
+    })
+
+    it('remembers task name, platform and timings when switching between workflows', () => {
+      // Check default options
+      cy
+        .get('.c-analysis table > tbody > tr')
+        .should('have.length', numTasks)
+        .should('be.visible')
+
+      // Set platform filter options
+      cy
+        .get('#c-analysis-filter-task-platforms')
+        .click({ force: true })
+      cy
+        .get('.v-list-item')
+        .contains('platform_1')
+        .click({ force: true })
+
+      // Set queue task name filter options
+      cy
+        .get('#c-analysis-filter-task-name')
+        .type('wait')
+
+      // Set task times filter options
+      cy
+        .get('#c-analysis-filter-task-timings')
+        .click({ force: true })
+      cy
+        .get('.v-list-item')
+        .contains('Queue')
+        .click({ force: true })
+
+      // Navigate away
+      cy.visit('/#/')
+      cy.get('.c-dashboard')
+      // Navigate back
+      cy.visit('/#/workspace/one')
+
+      // Check number of tasks
+      cy
+        .get('.c-analysis table > tbody > tr')
+        .should('have.length', 1)
+        .should('be.visible')
+    })
```

Comment:
This is not quite checking the right thing in this case, as the "No data available" placeholder displays as a row. I think you could add `contains('waiting')` to make sure it is right.

![image](https://github.com/cylc/cylc-ui/assets/61982285/1ab96ce6-3ce3-4529-8103-1a50078f145d)